### ISSUE TITLE
Add note that this is a Fabric mod on Curseforge page

### ISSUE THREAD

**Issue by TheDeviantCrafter** [InsomniaKitten/Couplings#21]

**Comment by frakier** [comment]
That seems like an important bit of information that should be on the curse page.
And it looked like such a promising mod.

**Comment by LambdAurora** [comment]
This is CurseForge fault's, not this mod's fault.

For a while CurseForge had a special category for Fabric because they didn't have any proper system for game loaders.
It was great to see if a mod was Fabric quickly.

Then they removed it because now they have a """proper""" system for those.
It just happens they haven't fixed search at all.

And if we look at CurseForge's "great" rules, we can see that projects are disallowed to contain the mod loader in their name (and even worse, multi-loader mods are forbidden to be multiple projects for each loader).

So, if you want to report this discoverability issue to someone, it's to CurseForge, it's been months and they still haven't fixed something as basic as selecting both a loader, and a version in their search system.

**Comment by TheDeviantCrafter** [comment]
Yeah, search is kind of a disaster right now. I was suggesting something like this:

![image](https://user-images.githubusercontent.com/18755309/154592846-e46280ce-a57f-4c35-b35c-ee0acb5a93eb.png)

**Comment by frakier** [comment]
Yeah like the author of the issue I was looking at the page... not search. On my mod's page I include the information so people know up front that it is forge by listing the forge version each version of mod requires... I have not learned fabric yet as I'm still working on forge. 
My point... I was on the mod's page and there was no information, so I came to github to see what its foundation was.
It does look like a promising mod.
